src/reactions/index.ts

```diff
@@ -0,0 +1,163 @@
+/*
+Copyright 2024 New Vector Ltd.
+
+SPDX-License-Identifier: AGPL-3.0-only
+Please see LICENSE in the repository root for full details.
+*/
+
+import { RelationType } from "matrix-js-sdk/src/types";
+
+import catSoundOgg from "../sound/reactions/cat.ogg?url";
```

Comment:
These should be preloaded to prevent a delay between the reaction and the sound.